chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -293,6 +293,9 @@ pane.item.related.count.singular = %S related:
 pane.item.related.count.plural  = %S related:
 pane.item.parentItem			= Parent Item:
 
+drag.invalid.title 				= Invalid item
```

Comment:
Capitalized: "Invalid Item" for a title

(Not sure if this merits its own string — every string creates extra localization work — but I don't know if there's an existing `general.` string that would work. "Error" seems a little strong.)